### PR TITLE
Revise LogLevel Enum

### PR DESCRIPTION
**Pull request by BrennanConroy**
#333

### CODE REVIEW

File: test/Microsoft.Extensions.Logging.Test/LoggerTest.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public void Log_IgnoresExceptionInIntermediateLoggersAndThrowsAggregateException
             // Assert
             Assert.Equal(new[] { "provider1.Test-Hello!", "provider3.Test-Hello!" }, store);
             Assert.NotNull(aggregateException);
-            Assert.Equal("An error occurred while writing to logger(s).", aggregateException.Message);
```
Comment (BrennanConroy) [inline test/Microsoft.Extensions.Logging.Test/LoggerTest.cs]: These are in preparation for newer CoreCLR packages that changed the Message overload of AggregateException https://github.com/dotnet/coreclr/commit/15db7abcb1aaec8f29f0029bc73632494dc281b3

File: src/Microsoft.Extensions.Logging.Abstractions/LogLevel.cs [hunk 2]
```diff
@@ -12,40 +12,40 @@ public enum LogLevel
...
         /// immediate attention.
         /// </summary>
-        Critical = 6,
+        Critical = 5,
 
         /// <summary>
         /// Not used for writing log messages. Specifies that a logging category should not write any messages.
         /// </summary>
-        None = int.MaxValue,
+        None = 6,
```
Comment (dougbu) [inline src/Microsoft.Extensions.Logging.Abstractions/LogLevel.cs]: Could you remind me: Why isn't `None` zero?

Comment (BrennanConroy) [inline src/Microsoft.Extensions.Logging.Abstractions/LogLevel.cs]: Nope, see #333

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**